src/react/deckgl.js

```diff
@@ -50,7 +51,8 @@ const defaultProps = {
   effects: [],
   onWebGLInitialized: noop,
   onLayerClick: noop,
-  onLayerHover: noop
+  onLayerHover: noop,
+  onLayerDrawn: noop
```

Comment:
I knew we once have onAfterRender and onBeforeRender but later deprecate it. I'm only add this function back for the test purposes. We need to exact the rendered canvas to check correctness. We shouldn't let normal developers to use this for feature development.

As for the name, I'm open to any suggestions. onAfterRender() seems to be better